### DISCUSSION TITLE
How to modify README.md?

### DISCUSSION THREAD

**Discussion by flexprojecthub** [community/community#147818] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Can I know how to use an image for README.md and how to animate it?

**Comment by Ravindulakmina** [comment, marked as answer]
There are several methods. I will show you the best ones that I use. we are using Markdown syntax.

```bash
![Alt text](image-url)
```    
You can provide the URL of where the image you want is hosted instead of the image URL.
You can provide images in formats like gif, png, jpg here.Even animated gifs can be provided.

EX : 
```bash
![Logo](https://example.com/path-to-image.png)
```
The best way to learn more about this is to read the README.md on my profile and the free README.md on other public accounts. You can learn a lot of useful tips and tricks.

If you have any questions, please ask.

**Reply by flexprojecthub** [reply]
I'll try. Thank you for the good answer.

**Reply by Ravindulakmina** [reply]
welcome. You can mark this as an answer if this is important to you.
